meta/main.yml

```diff
@@ -1,32 +1,28 @@
----
 galaxy_info:
-  author: Ken Rimey
-  description: Install the Stackdriver agent and configure its plugins.
+  author: Ryan Moriarty
+  description: Install the Cloud Ops Monitoring Agent
   company: Google
-  license: Apache
-  min_ansible_version: 1.7
+  license: Apache-2.0
+  min_ansible_version: 2.1
   platforms:
-    # Red Hat and derivatives
-    - name: EL
-      versions:
-        - 6
-        - 7
-    - name: Amazon
-      versions:
-        - 2014.03
-    # Debian and derivatives
     - name: Debian
       versions:
-        - wheezy
-        - jessie
+        - stretch
+        - buster
     - name: Ubuntu
       versions:
-        - trusty
-    # Windows
-    - name: Windows
+        - xenial
+        - bionic
+        - focal
```

Comment:
The monitoring agent supports groovy as well. I have a change under review to add it to the supported list.

Reply:
Done.